README.md

```diff
@@ -64,9 +64,36 @@ jobs:
       - uses: actions/checkout@v4
 
       - name: Run commitlint
+        id: commitlintrun
         uses: opensource-nepal/commitlint@v0.2.1
+        with:
+          fail_on_error: false
+
+      - name: Check Output
+        run: |
+            echo 'Status - ${{ steps.commitlintrun.outputs.status }}'
+            echo 'Exit Code - ${{ steps.commitlintrun.outputs.exit_code }}'
 ```
 
+#### Github Action Inputs
+
+1. **Proceed on Error** (fail_on_error)
```

Comment:
suggestion: Let's use the table format for the inputs and outputs.

Reply:
That's a nice suggestion, would incorporate it.